resources/authgear/templates/en/web/authflowv2/__alert_message.html

```diff
@@ -1,15 +1,18 @@
 {{ define "authflowv2/__alert_message.html" }}
-<div class="alert-message
-  {{ if eq $.Type "error" }}alert-message--error{{ end }}
-  {{ if eq $.Type "success" }}alert-message--success{{ end }}
-  {{ if eq $.Type "warning" }}alert-message--warning{{ end }}
-  {{ $.Classname }}
-">
-  <span class="alert-message__icon material-icons material-icons--filled">
-    {{ if eq $.Type "error" }}error{{ end }}
-  </span>
-  <span class="alert-message__text">
-    {{ .Message }}
-  </span>
-</div>
+{{ $message := trimSpace .Message }}
```

Comment:
I think `trimSpace` is not needed, because you can use `{{- -}}` to trim the spaces.

Reply:
Yes.

Reply:
Per discussion in huddle we will replace the existing trim function.
https://github.com/authgear/authgear-server/pull/3660#discussion_r1462705337